projects/ngx-formentry/src/components/ngx-remote-select/ngx-remote-select.component.ts

```diff
@@ -34,6 +34,7 @@ export class RemoteSelectComponent implements OnInit, ControlValueAccessor {
   selectedRemoteOptions: SelectOption;
   @Input() placeholder = 'Search...';
   @Input() componentID: string;
+  @Input() disabled: Boolean = false;
```

Comment:
`boolean` (but again, please drop the type - you have it initialized to boolean already)

Reply:
Should be sorted now